src/main/org/nlogo/nvm/LetBinding.java

```diff
@@ -4,9 +4,9 @@
 
 import org.nlogo.api.Let;
 
-strictfp class LetBinding {
-  final Let let;
-  Object value;
+strictfp public class LetBinding {
+  public final Let let;
+  public Object value;
```

Comment:
@TheBizzle , what were you saying about grad students going around making all the fields public....

In this case I would prefer public accessor methods for anything outside of nvm, but I'm fine the with the fields being package-private.